rero_ils/modules/items/serializers/json.py

```diff
@@ -119,8 +128,3 @@ def post_process_serialize_search(self, results, pid_fetcher):
             filter_document_type_buckets(buckets)
 
         return super().post_process_serialize_search(results, pid_fetcher)
-
-        # Correct document type buckets
```

Comment:
If you delete this the facette for the documents are not displayed correctly.

Reply:
this code was after the return. It is also before the return. Code cleaning.

Reply:
You are right this code exist also in line 116.